src/main/java/org/skriptlang/skript/bukkit/recipes/elements/ExprRecipeExperience.java

```diff
@@ -0,0 +1,96 @@
+package org.skriptlang.skript.bukkit.recipes.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.CookingRecipe;
+import org.bukkit.inventory.Recipe;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.recipes.MutableRecipe;
+import org.skriptlang.skript.bukkit.recipes.MutableRecipe.MutableCookingRecipe;
+import org.skriptlang.skript.bukkit.recipes.CreateRecipeEvent;
+
+@Name("Recipe Experience")
+@Description("The experience of a blasting, furnace, campfire, or smoking recipe.")
+@Examples({
+	"set {_recipe} to a new blasting recipe with the key \"my_recipe\":",
+		"\tset the recipe input item to a raw copper named \"Impure Copper\"",
+		"\tset the recipe experience to 20",
+		"\tset the recipe result to copper ingot named \"Pure Copper\""
+})
+@Since("INSERT VERSION")
+public class ExprRecipeExperience extends PropertyExpression<Recipe, Float> {
```

Comment:
we might want to use the experience type here
though I do know it can be awkward for the user, so let me know if you think it's better to leave it as a number

Reply:
I  could do both, of just the experience type

Let me know

Reply:
eh just leave it, I'm not sure if we should even have the experience type tbh